Bug: Pre-commit check_commits failing due to empty ident name

https://github.com/rust-bitcoin/rust-lightning/pull/691/checks?check_run_id=1200685353

```
Run git remote add upstream https://github.com/rust-bitcoin/rust-lightning
From https://github.com/rust-bitcoin/rust-lightning
 * [new branch]      main       -> upstream/main
 * [new branch]      master     -> upstream/master
Rebasing (1/45)
*** Please tell me who you are.

Run

  git config --global user.email "you@example.com"
  git config --global user.name "Your Name"

to set your account's default identity.
Omit --global to set the identity only in this repository.

fatal: empty ident name (for <runner@fv-az120.rmsfdau0lhjerfqozzp3rn115f.bx.internal.cloudapp.net>) not allowed
Error: Process completed with exit code 128.
```

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -109,6 +109,8 @@ jobs:
         run: |
           git remote add upstream https://github.com/rust-bitcoin/rust-lightning
           git fetch upstream
+          export GIT_COMMITTER_EMAIL="rl-ci@example.com"
+          export GIT_COMMITTER_NAME="RL CI"
           git rebase upstream/main
       - name: For each commit, run cargo check (including in fuzz)
         run: ci/check-each-commit.sh upstream/main

